base/MAPL_GridManager.F90

```diff
@@ -457,8 +457,10 @@ function make_factory_from_file(this, file_name, unused, force_file_coordinates,
          attr => file_metadata%get_attribute('grid_type')
          attr_value => attr%get_value()
          select type (attr_value)
-         type is (StringWrap)
-            grid_type => attr_value%value
+         type is (character(*))
+            grid_type => attr_value
+         class default
+            _FAIL("grid_type attribute must be stringwrap") 
```

Comment:
We might still need the StringWrap  case for gfortran?  Are you sure this can be changed?   I guess if it passes CI ...

Reply:
It seems to have passed CI, but all that means is that GNU can compile it. Maybe @bena-nasa knows if any of our unit tests or integration tests would hit this code?

Reply:
@I'll try running Regrid_Util.x on this branch with gfotran with tripolar input or output to exercise this and see.